frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -53,6 +52,9 @@ export const dashboardLogic = kea<dashboardLogicType>({
         setAutoRefresh: (enabled: boolean, interval: number) => ({ enabled, interval }),
         setRefreshStatus: (id: number, loading = false) => ({ id, loading }), // id represents dashboardItem id's
         setRefreshError: (id: number) => ({ id }),
+        setPageTitle: ({ title }) => {
+            document.title = title ? `${title} • PostHog` : 'PostHog'
```

Comment:
Almost there, it's just that actions in Kea need to be pure functions, so you need to actually pass the attribute name as we do with actions above and instead set up a `listener` with the same name as the action that actually executes this code.

Reply:
🤦‍♀️  thanks